scripts/concurrent_task_utils.py

```diff
@@ -42,13 +44,52 @@ def log(message, show_time=False):
             python_utils.PRINT(message)
 
 
+class TaskResult(python_utils.OBJECT):
+    """Task result for concurrent_task_utils."""
+
+    def __init__(self, name, failed, trimmed_messages, full_messages):
+        """Constructs a TaskResult object.
+
+        Args:
+            name: str. The name of the task.
+            failed: bool. The boolean value representing whether the task
+                failed.
+            trimmed_messages: list(str). List of error messages that are
+                trimmed to keep main part of messages.
+            full_messages: str|list(str). List of full messages returned by the
```

Comment:
full_messages should be of one type, change it in the backend test to return a list or change it in linters to return a string* (I would prefer the first one, as the name of the variable is plural.)